setup.cfg

```diff
@@ -0,0 +1,43 @@
+[metadata]
+name = enetpy
+author = Thermondo GmbH
+author-email = opensource@thermondo.com
+summary = Python wrapper for the ene't Marktpartner API
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8
+home-page = https://github.com/Thermondo/python-enet
+project_urls =
+    Bug Tracker = https://github.com/Thermondo/python-enet/issues
+    Documentation = https://github.com/Thermondo/python-enet
+    Source Code = https://github.com/Thermondo/python-enet
+license = MIT License
+classifier =
+    Development Status :: 5 - Production/Stable
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache-2
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Framework :: Django
```

Comment:
Does it have to do something with Django now? Or are you planning to add Django-related functionality?